dts/arm/atmel/same70.dtsi

```diff
@@ -430,7 +431,8 @@
 
 		can1: can@40034000 {
 			compatible = "atmel,sam-can";
-			reg = <0x40034000 0x100>;
+			reg = <0x40034000 0x100>, <0x40088114 0x4>;
+			reg-names = "m_can", "dma_base_addr";
```

Comment:
`_addr` is kind of implied here? How about just naming it (and the corresponding variable) `dma_base`?